README.md

```diff
@@ -1,11 +1,18 @@
 ### appliance-os
 
-Ansible playbooks to configure a Kubernetes cluster.
+Ansible playbooks to configure a Kubernetes cluster node/master.
 
-Edit the [hosts.yml](hosts.yml) file with the correct IPs and users.
 
-Run the playbook:
```

Comment:
Could you add the following:

Start Multipass machine locally:
```
multipass launch -d 16G -m 4G -n os-builder -c 4
```
Paste your public key in:
```
multipass exec os-builder nano .ssh/authorized_keys
```
Retrieve the machine IP and set it in [hosts.ini](inventory/sample/hosts.ini)
```
multipass info os-builder | grep IPv4
```